File: 1. Two Sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        j = len(nums) - 1
        while True:
            for k in range(0, j):
                if nums[j] + nums[k] == target:
                    return [k, j]
            j = j - 1
            k = 0

File: 1. Two Sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        int[] idx = new int[2];
        int i = 0, j;
        for (i = 0; i < nums.Length - 1; i++){
            for (j = i + 1; j < nums.Length; j++){
                if (nums[i] + nums[j] == target){
                    idx[0] = i;
                    idx[1] = j;
                    return idx;
                }
            }
        }
        return idx;
    }
}
